### DISCUSSION TITLE
This request was automatically failed because there were no enabled runners online to process the request for more than 1 days.

### DISCUSSION THREAD

**Discussion by aejuice-github** [community/community#28226] [Actions]
I have a self-hosted runner and long build times. I add 50-100 jobs to the queue. Each job can be 5-10 hours.
Jobs fail before the runner can take them. The limitation is ridiculous. I specified timeout-minutes: 144000
which is 10 days. I will have to build a custom job queue and move away from GitHub because of this issue. Is there a way to actually set up a timeout longer than 1 day??
P.S. I'm referring to Job queue time which is hardcode limited to 1 day according to documentation.https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#usage-limits

**Comment by lukesneeringer** [comment]
I have more or less the exact converse issue. I have a job that runs every night and takes minutes, but which is important and I want to know if it fails much quicker if runners go down and fail to pick it up. It would be really nice to be able to set this to a lower (or higher) value.

**Comment by Wenzel** [comment]
:up: 
Facing the exact same problem here, 300 jobs scheduled in the queue, crunched with 10 workers at a time.
As each job takes 1h to complete, my CI fails because of this artificial limitation.

It's been 2 years since this has been first posted, and no answers from Github Actions team.. ?

**Comment by telnettrauma** [comment]
I only have 4 jobs, all of which are Dependabot, and I still get this error! I still don't know if it's supposed to be doing this or not. Very annoying, I wish there was just **some** clarification.
